www/scripts/codecheckerviewer/CheckerStatistics.js

```diff
@@ -0,0 +1,204 @@
+//                     The CodeChecker Infrastructure
+//   This file is distributed under the University of Illinois Open Source
+//   License. See LICENSE.TXT for details.
+// -------------------------------------------------------------------------
+
+define([
+  'dojo/_base/declare',
+  'dojo/data/ItemFileWriteStore',
+  'dojo/Deferred',
+  'dojo/promise/all',
+  'dojo/store/Memory',
+  'dojo/store/Observable',
+  'dojox/form/CheckedMultiSelect',
+  'dojox/grid/DataGrid',
+  'dijit/layout/ContentPane',
+  'codechecker/hashHelper',
+  'codechecker/util'],
+function (declare, ItemFileWriteStore, Deferred, all, Memory, Observable,
+  CheckedMultiSelect, DataGrid, ContentPane, hashHelper, util) {
+  var RunFilter = declare(CheckedMultiSelect, {
+    _updateSelection: function() {
+      this.inherited(arguments);
+      if(this.dropDown && this.dropDownButton){
+        var label = '';
+
+        this.options.forEach(function(option) {
+          if (option.selected)
+            label += (label.length ? ', ' : '') + option.label;
+        });
+
+        this.dropDownButton.set('label', label.length ? label : this.label);
+      }
+    }
+  });
+
+  var FilterPane = declare(ContentPane, {
+    constructor : function () {
+      var that = this;
+
+      this._runStore = new Observable(new Memory({}));
+
+      this._runFilter = new RunFilter({
+        store: this._runStore,
+        labelAttr : 'label',
+        label : 'Get Statistics only for these runs...',
+        multiple : true,
+        dropDown : true,
+        onChange: function (state) {
+          that.selectedRuns = state;
+          that.dataGrid.refreshGrid(state);
+          hashHelper.setStateValue('run', state);
+        }
+      });
+
+      this.selectedRuns = null;
+    },
+
+    postCreate : function () {
+      this.addChild(this._runFilter);
+
+      var state = hashHelper.getValues();
+      if (state.run)
+        this.selectedRuns = state.run instanceof Array
+          ? state.run.map(function (run) { return parseInt(run); })
+          : [parseInt(state.run)];
+
+      this.loadRunStoreData();
+    },
+
+
+    loadRunStoreData : function () {
+      var that = this;
+
+      CC_SERVICE.getRunData('', function (runs) {
+        runs.sort(function (a, b) {
+          if (a.name > b.name) return 1;
+          if (a.name < b.name) return -1;
+
+          return 0;
+        }).forEach(function (run) {
+          that._runStore.put({id : run.runId, label : run.name});
+        });
+
+        that._runFilter.set('value', that.selectedRuns);
+      });
+    }
+  });
+
+  var CheckerStatistics = declare(DataGrid, {
+    constructor : function () {
+      this.store = new ItemFileWriteStore({
+        data : { identifier : 'id', items : [] }
+      });
+
+      this.structure = [
+        { name : 'Checker Name', field : 'checker', width : '100%'},
+        { name : 'Severity', field : 'severity', styles : 'text-align: center;', width : '10%'},
```

Comment:
Please make this column a bit bigger, so on relatively medium-sized screens the `Severity` header is not split.